hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalScheduledExecutorStatsImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.hazelcast.monitor.impl;
+
+import com.hazelcast.internal.json.JsonObject;
+import com.hazelcast.internal.metrics.Probe;
+import com.hazelcast.monitor.LocalScheduledExecutorStats;
+import com.hazelcast.scheduledexecutor.impl.ScheduledTaskDescriptor;
+
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class LocalScheduledExecutorStatsImpl implements LocalScheduledExecutorStats {
+
+    @Probe
+    private final AtomicLong scheduledSingleRun = new AtomicLong();
+    @Probe
+    private final AtomicLong scheduledFixedRate = new AtomicLong();
+    public volatile ConcurrentMap<String, ScheduledTaskDescriptor> tasks;
```

Comment:
Do we need to reference the tasks an expose them through metrics? Whats the use-case?
Keep in mind that the tasks in this map can be just backup entries on a partition replica.

Reply:
The use case is adding metrics for debugging purposes. Not exposing statistics which can be accessed from Java.